### ISSUE TITLE
Accessibility Bug: The items from the ‘Drop-down popover’ is not reachable via keyboard.

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3144]
Fundamental core component: "Popover"

Description: The items from the ‘Drop-down popover’ is not reachable via keyboard.

Expected: All the items within the ‘Dropdown popover’ should be able to reach via keyboard.

Attached screenshot:

![image](https://user-images.githubusercontent.com/32538291/91651485-0147a080-eaab-11ea-8837-69b7497141c4.png)

**Comment by Sanjay-1312** [comment]
Hello Colleagues,

The issue is fixed.
hence i am closing the ticket.

Thanks and Regards
Sanjay
